tests/test_lucxor_regression.py

```
@@ -456,110 +456,71 @@ class TestEndToEndRegression:
+        output2 = str(tmp_path / "output2.idparquet")
+        runner = CliRunner()
+
+        args_base = [
+            "lucxor",
+            "--input-spectrum", mzml,
+            "--input-id", idparquet,
+            "--target-modifications", "Phospho(S),Phospho(T),Phospho(Y)",
+        ]
+
+        for out in [output1, output2]:
+            r = runner.invoke(cli, args_base + ["--output", out])
```

Ruff (0.15.15)

[warning] Consider `[*args_base, "--output", out]` instead of concatenation

Replace with `[*args_base, "--output", out]`

(RUF005)